package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "chartmogul-node",
-  "version": "1.3.0",
+  "version": "2.0.0",
```

Comment:
There are no breaking changes here. But the version is bumped for consistency with other libraries.